README.md

```diff
@@ -17,60 +15,66 @@ You can install a specific version by adding the tag at the end, e.g, `pip insta
 
 ## Usage
 
+1. After installing the plugin run:
+
 ```bash
 tutor plugins enable distro
+```
+
+2. Configure Distro. The plugin manages a set of settings that you can add, refer to the following documentation to know the Distro variables:
+
+- [How to customize distro](./docs/how_to_customize_distro.rst)
+- [How to add a new package](./docs/how_to_add_new_packages.rst)
+
+:warning: Remember that from version 15, Distro plugin has no default values.
+
+3. Once you add the configuration, you can access the next commands (use it depending on your use case, more info below).
 
-# Validator commands for config file
+```bash
+# Validate the Distro settings are properly set
 tutor distro syntax-validator
+# Validate the repositories for packages are valid
 tutor distro repository-validator
 
-# Enabler commands
+# Enable the theme(s) defined
 tutor distro enable-themes
+# Enable the package(s) defined
 tutor distro enable-private-packages
 
-# Run extra commands
+# Run Tutor commands
 tutor distro run-extra-commands
 ```
 
-### Documentation
-
-Distro plugin manages a set of settings that you can configure, to know how to do that check:
-
-- [How to customize distro](./docs/how_to_customize_distro.rst)
-- [How to add a new package](./docs/how_to_add_new_packages.rst)
+4. Build the custom docker image running `tutor images build openedx` or `tutor images build openedx-dev`.
 
-# Required tutor settings
+5. Launch your customized instance `tutor local launch` or `tutor dev launch`.
 
-Define the docker images to be used and build it before starting the instance to avoid issues.
-Example:
-
-```yaml
-DOCKER_IMAGE_OPENEDX: "docker.io/ednxops/distro-edunext-edxapp:quince"
-DOCKER_IMAGE_OPENEDX_DEV: "docker.io/ednxops/distro-edunext-edxapp-dev:quince"
-```
+**Notes:**
 
-Also, you need an edx-platform version distro compatible.
+- If you want to use a custom edx-platform, you will need to create a branch based on the release, you can see the compatibility table below for reference:
 
-| openedx | distro   | tutor |
-| ------- | -------- | ----- |
-| lilac   | limonero | v12   |
-| maple   | mango    | v13   |
-| nutmeg  | nuez     | v14   |
-| olive   | olmo     | v15   |
-| palm    | palma    | v16   |
-| quince  | quince   | v17   |
+| openedx | tutor |
+| ------- | ----- |
+| lilac   | v12   |
+| maple   | v13   |
+| nutmeg  | v14   |
+| olive   | v15   |
+| palm    | v16   |
+| quince  | v17   |
 
-:warning: **NOTE**: From Olmo version Distro has not defaulted packages. Now it is necessary to add the packages you want in the ``config.yml`` file. See [How to add a new package](./docs/how_to_add_new_packages.rst)
+- Define the docker images to be used and build it before starting the instance to avoid issues.
```

Comment:
```suggestion
- Specify the docker images for your installation and build them before starting before starting the instance to avoid issues with missing images.
```

Reply:
I wonder if this happens without manually building the image before launching